src/utils/trigger-column-utils.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { Label, Truncate, Flex, FlexItem } from '@patternfly/react-core';
+import { CommitIcon } from '~/components/Commits/CommitIcon';
+import { PipelineRunEventType } from '../consts/pipelinerun';
+import { ExternalLink } from '../shared';
+
+export interface TriggerColumnData {
+  gitProvider?: string;
+  repoOrg?: string;
+  repoURL?: string;
+  prNumber?: string;
+  eventType?: string;
+  commitId?: string;
+}
+
+/**
+ * Component that renders trigger column data with appropriate icons and links
+ * @param props - Object containing trigger-related data
+ * @returns JSX element for the trigger column
+ */
+export const TriggerColumnData: React.FC<TriggerColumnData> = ({
```

Comment:
how about moving this component to `src/components` or `src/shared/components` instead? 🤔